ckanext/dia/plugin/__init__.py

```diff
@@ -62,6 +62,15 @@ def update_config(self, config):
         tk.add_template_directory(config, '../templates')
 
 
+class DIAAPIPlugin(p.SingletonPlugin):
+    p.implements(p.IBlueprint)
+
+    # IBlueprint
+
+    def get_blueprint(self):
+        return views.api
```

Comment:
This just registers a "plugin" that registers the Flask blueprint so that CKAN's routing knows about it.